md/installation_guide.md

```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
 
 Then when using the installer provide this generated file as a keystore, the same password for the keystore and the key, and the same alias.
 
+#### Renew certificate
+
+When the certificate expires, it can be renewed by reinstalling the backend (not the storage).
+
+* stop the backend using `bonita-ici stopApp`
+* start and deploy the backend using either the new certificate or newly generated one using `bonita-ici startApp deploy`
```

Comment:
What is the difference between "new certificate" and "newly generated one"?
We can talk maybe. I'm missing something... :/

Reply:
`generated one` is the one that the installer generate if you choose to do so, and `new certificate` if you provided one

Reply:
ok. I think it can be clarified, but again, not a big deal.